src/infra/queue/adapter/bull.js

```diff
@@ -35,7 +35,9 @@ export default class BullAdapter {
     const options = {
       priority,
       delay,
-      timeout: ttl
+      timeout: ttl,
+      attempts: 3,
```

Comment:
I agree, that's what I've prepared today:
https://github.com/hull-ships/hull-intercom/pull/55/files#diff-1cfe8e7411521eb568edc55dd0503c1b